index.md

```diff
@@ -22,160 +22,877 @@ The following projects are included as part of the Data and Visualization SDK:
+  animation and analysis tool.
+* [VTK-m][VTK-m]: A toolkit if scientific visualization algorithms for emerging
+  processor architectures.
+
+[Ascent]: https://github.com/Alpine-DAV/ascent
+[Cinema]: https://cinemascience.github.io
+[ParaView]: https://paraview.org
+[SENSEI]: https://sensei-insitu.org/
+[VisIt]: https://visit-dav.github.io/visit-website/
+[VTK-m]: https://m.vtk.org
 
 ### Compression
 
-* [SZ](https://szcompressor.org): An error-bounded lossy data compressor for
-  floating-point and integer datasets.
-* [ZFP](https://computing.llnl.gov/projects/zfp): An open-source library for
-  compressed floating-point arrays that support high throughput read and write
-  random access.
-
-## Current project integration status
-
-#### Key
-
-* CPU: If the spack package exists for CPU-only features
-  * \- : None
-  * 0 : Broken
-  * 1 : Confirmed working
-* CUDA: If the spack package supports NVIDIA CUDA
-  * \- : Not applicable to this package
-  * 0 : Broken
-  * 1 : Confirmed working
-* ROCm: If the spack package supports AMD ROCm
-  * Same options as for CUDA
-* SDK CPU: If the package is enabled in the SDK without GPU support
-  * 0 : Broken when enabled in the SDK
-  * 1 : Can only be enabled with some of the SDK packages while conflicting with
-    others
-  * 2 : Can be enabled with all the other SDK packages
-* SDK CUDA: If the package is enabled in the SDK with NVIDIA CUDA enabled
-  * \- : Not applicable to this package
-  * All other options for SDK CPU
-* SDK ROCm: If the package is enabled in the SDK with AMD ROCm enabled
-  * Same options as for SDK CUDA
-
-### General Spack support
-
-| Project  | CPU | CUDA | ROCm | SDK CPU | SDK CUDA | SDK ROCm |
-|:--------:|:---:|:----:|:----:|:-------:|:--------:|:--------:|
-| ADIOS    | 1   | -    | -    | 2       | -        | -        |
-| Darshan  | 1   | -    | -    | 2       | -        | -        |
-| HDF5     | 1   | -    | -    | 2       | -        | -        |
-| PNetCDF  | 1   | -    | -    | 2       | -        | -        |
-| UnifyFS  | 1   | -    | -    | 2       | -        | -        |
-| VeloC    | 1   | -    | -    | 2       | -        | -        |
-| Ascent   | 1   | 1    | 1    | 2       | 1 *      | 1 *      |
-| Cinema   | 1   | -    | -    | 2       | -        | -        |
-| ParaView | 1   | 0 *  | -    | 2       | 0 *      | -        |
-| SENSEI   | 1   | -    | -    | 0 *     | -        | -        |
-| VisIt    | 1   | -    | -    | 0 *     | -        | -        |
-| VTK-m    | 1   | 1    | 1    | 2       | 1 *      | 1 *      |
-| SZ       | 1   | -    | -    | 2       | -        | -        |
-| ZFP      | 1   | 1    | -    | 2       | 2        | -        |
-
-#### Notes
-{:.no_toc}
+* [SZ][SZ]: An error-bounded lossy data compressor for floating-point and
+  integer datasets.
+* [ZFP][ZFP]: An open-source library for compressed floating-point arrays that
+  support high throughput read and write random access.
```

Comment:
The diff makes things look strange and confusing, each previous `Notes` section should have been converted into footnotes for a given table correctly.  Please double check the original notes are linked correctly in the right locations, some may be stale / should get removed instead?